content/en/docs/setup/pick-right-solution.md

```diff
@@ -72,13 +72,13 @@ These solutions allow you to create Kubernetes clusters on a range of Cloud IaaS
 few commands. These solutions are actively developed and have active community support.
 
 * [Conjure-up Kubernetes with Ubuntu on AWS, Azure, Google Cloud, Oracle Cloud](/docs/getting-started-guides/ubuntu/)
-* [Google Compute Engine (GCE)](/docs/getting-started-guides/gce/)
-* [AWS](/docs/getting-started-guides/aws/)
-* [Azure](/docs/getting-started-guides/azure/)
+* [Google Compute Engine (GCE)](/docs/setup/turnkey/gce/)
+* [AWS](docs/setup/turnkey/aws/)
+* [Azure](/setup/turnkey/azure/)
```

Comment:
this absolute path is incorrect?

Reply:
Yes， but seems what i change also incorrect:https://deploy-preview-9256--kubernetes-io-master-staging.netlify.com/docs/setup/pick-right-solution/. may be fix later